package.json

```diff
@@ -45,6 +45,7 @@
     "typescript": "2.1.5"
   },
   "dependencies": {
+    "cloud-device-emulator": "0.1.0",
```

Comment:
Do you think it will be a good idea to use `~0.1.0` instead of `0.1.0`? I think it will be good to get bugfixes from `cloud-device-emulator` without releasing new version of `nativescript-cloud`.